src/main/controllers/BaseSearchController.ts

```diff
@@ -112,9 +112,21 @@ export abstract class BaseSearchController<SearchType extends SearchTypeCommon>
   abstract post(req: AppRequest, res: Response): Promise<void>;
 
   formatSearchRequest(request: Partial<SearchType>): void {
-    // Remove any properties with empty strings from the request object
-    // @ts-ignore
-    Object.keys(request).forEach(key => request[key] === '' ? delete request[key] : {});
+  // Remove any properties with empty strings from the request object
+    Object.keys(request).forEach(key => {
+      const typedKey = key as keyof SearchType;
+      if (request[typedKey] === '') {
+        delete request[typedKey];
```

Comment:
Do we pass empty strings as part of the search

Reply:
Also If I remove this code all the unit test start failing as at the moment it's only passing the parameters where we don't have empty string , for example all the test case which added as part of this and any previous one